process/tasks/ABProcessTaskServiceInsertRecordCore.js

```diff
@@ -107,6 +109,25 @@ module.exports = class InsertRecordCore extends ABProcessElement {
       return obj.fields((f) => f.id == this.repeatColumn)[0];
    }
 
+   /**
+    * processDataFields()
+    * return a single available data field from this element
+    * this will be the record inserted by this task
+    * Different Process Elements can make data available to other
+    * process Elements.
+    * @return {array} | null
+    */
+   processDataFields() {
+      return [
```

Comment:
the priority right now is to get the PK returned, I haven't figured out how to return the other fields.

iirc....
The abdesigner uses the uuids of the fields, the object I have after creating the record is more like `fieldLabel: fieldData`
need access to the object definition to convert between the two....

It seems like it would be better for a full implementation to use `ABQL` classes (or at least that kind of structure), as those are FOR "storing process task data for use in other elements"

Problem is, I can't think of a use case where I want anything but the new PK from insert task 🤔

Reply:
This only puts the `insert task` into the UI